how-to/divio-app.rst

```diff
@@ -0,0 +1,126 @@
+.. _divio-app:
+
+How to install and use the Divio app
+====================================
+
+The Divio app helps you run your Divio projects locally in our Docker environment, upload and download media, database
+and configuration changes as well as set up a shell environment ready for interaction with the project running in the
+Docker container.
+
+
+Set up the Divio app
+--------------------
+
+
+Installation
+~~~~~~~~~~~~
+
+* **Download** the `Divio app here <https://divio.com/developers/#app>`_, which is available for Macintosh, Linux and
+  Windows.
+
+* Open the downloaded package and follow the on-screen instructions to install the application.
+
+.. _note:: 
+
+  Some anti-virus software for Windows may warn that the Divio app application is unsigned and therefore represents a
```

Comment:
is this true? we do correctly sign the app. I've seen instances of "we prevented the app from running cause it's from unknown publisher" but if you click on more info it would show that it is signed by `Divio Technologies AB` 

the unknown publisher warning goes away with sufficient installs

Reply:
and it's safe to ignore the "unknown publisher" warning, if it says it is _unsigned_ then it's not safe

Reply:
I removed it, because I could not verify it.